roles/control_node/tasks/20_all_workshop.yml

```diff
@@ -20,6 +32,12 @@
     owner: "{{ username }}"
     group: "{{ username }}"
 
+- name: Apply patch for runner fix PR759
+  lineinfile:
+    path: /usr/lib/python3.8/site-packages/ansible_runner/config/_base.py
+    regexp: ".*--userns=keep-id.*"
+    state: absent
+
```

Comment:
I agree with this as a temp fix. It'll have to be removed once bug is fixed

Reply:
+1